dist/index.js

```diff
@@ -9677,17 +9719,20 @@ const pantheonDeploy = (() => {
             await exec.exec('terminus', ['auth:login', `--machine-token=${ machineToken }`]);
             await exec.exec('terminus', ['multidev:create', pantheonRepoName, pullRequest.head.ref]);
 
-            output = JSON.stringify(child_process.execSync(`terminus env:view --print ${ pantheonRepoName }.${ pullRequest.head.ref }`));
+            const output = JSON.stringify(child_process.execSync(`terminus env:view --print ${ pantheonRepoName }.${ pullRequest.head.ref }`));
+            console.log('URL to access the multidev is : ' . output);
```

Comment:
"\n " in the console log

Reply:
Yeah they're everywhere.. I think they are needed if we want them to be properly displayed ?